jupyter-docker/Dockerfile

```diff
@@ -203,17 +215,11 @@ RUN apt-get update \
     libxml2-dev
 
 RUN R -e 'install.packages(c( \
-    "repr",  \
```

Comment:
These changes seem to be from the tidyverse PR? Might be best to revert the R stuff from this PR to keep them separate (and hopefully avoid merge conflicts).